pmprophet/model.py

```diff
@@ -7,7 +7,19 @@
 
 
 class PMProphet:
-    def __init__(self, data, growth=False, intercept=True, model=None, name=None, change_points=[], n_change_points=0):
+    """Prophet forecaster.
+
+    Parameters
+    ----------
+    data: Data to be used for fitting the model.
```

Comment:
I like numpy doc-strings: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html (missing indentation, data-type and indented new-line for var description.

Reply:
@twiecki usually I do, here just a draft copied (with edits) to great extent from fbprophet that used such form. Much can be improved, but I'm looking for comments not to force anything.

Reply:
I agree with @twiecki on this, much clearer